bindings/rust/s2n-tls-sys/Cargo.toml

```diff
@@ -31,3 +31,10 @@ openssl-sys = { version = "0.9" }
 
 [build-dependencies]
 cc = { version = "1.0", features = ["parallel"] }
+cmake = { version = "0.1", optional = true }
+# Pin to this version until s2n-tls supports OpenSSL 3.0
+# Build the vendored version to make it easy to test in dev
```

Comment:
Why do we need to pin to a version if the actual openssl version is automatically discovered?

Reply:
when you use the `vendored` feature flag it pulls the source and builds it locally for you. after `0.9.68` the pull openssl 3.0 instead of 1.1.1.

Reply:
but we only do this in dev to make it so you don't have to run our install scripts